teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>See comment on <code>SeqRecord</code> and <code>Seq</code> in the import cell.</p><p> </p><p>For pandas DataFrames, could you please use the following signature</p><pre class="ql-syntax" spellcheck="false">row["Sequence"]
</pre><p>instead of </p><pre class="ql-syntax" spellcheck="false">row.Sequence
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Changed. Seq.IO needs to be imported as Seq_IO</p>